src/promptflow-devkit/promptflow/_sdk/entities/_trace.py

```diff
@@ -177,6 +177,23 @@ def _to_rest_object(self) -> typing.Dict:
             "external_event_data_uris": rest_external_event_data_uris,
         }
 
+    def to_dict(self) -> typing.Dict:
+        """Return a dictionary that follows OpenTelemetry span spec."""
+        # manually build this dict, referring to `ReadableSpan.to_json` in OTel Python SDK
+        return {
+            "name": self.name,
+            "context": copy.deepcopy(self.context),
+            "kind": self.kind,
+            "parent_id": self.parent_id,
+            "start_time": self.start_time.isoformat(),
```

Comment:
ReadableSpan may have a "z" suffix, need to make sure this can be carried.

Reply:
https://github.com/open-telemetry/opentelemetry-python/blob/b51a6f8e62a10a4f22455f55e439fe5c5fcac44d/opentelemetry-sdk/src/opentelemetry/sdk/util/__init__.py#L24

Reply:
thanks for pointing this out! just updated how we get time string.